scripts/alloyBuilder.js

```diff
@@ -247,8 +247,8 @@ const getMakeBuildCommand = () => {
 };
 
 const getPushNotificationsServiceWorkerBuildCommand = () =>
-  new Command("build-pushnotifications-sw")
-    .description("Build the push notification service worker.")
+  new Command("build-sw")
+    .description("Build the s ervice worker.")
```

Comment:
```suggestion
    .description("Build the service worker.")
```

Reply:
I know, I know. I was thinking similarly. But Jon pointed out that we can change the entrypoint later without affecting the commands. It is something we can do behind the scenes at a later point without breaking anything.